content/10.methods.md

```diff
@@ -43,23 +43,41 @@ For other keynote speakers, we gleaned their full name as well as the year in wh
+
+We evaluated two resources for extracting corresponding authors from papers: [PubMed](https://pubmed.ncbi.nlm.nih.gov/) and [PubMed Central](https://www.ncbi.nlm.nih.gov/pmc/) (PMC).
+Both resources are provided by the U.S. National Library of Medicine and index scholarly articles.
+PubMed contains a record for every article published in journals it indexes (30 million records total circa 2020) and provides abstracts but not fulltext.
+PMC, which provides fulltext access, does not contain every article from every journal (5.9 million records total circa 2020).
+In general, open access journals will deposit their entire catalog to PMC (e.g. _BMC Bioinformatics_ & _PLOS Computational Biology_), while toll access journals (e.g. _Bioinformatics_) will only deposit articles when funders require it.
+Since PMC requires publishers to submit fulltext articles in a structured XML format, the machine-readability and breadth of metadata in PMC is often superior to PubMed.
+
+Of PMC's 5.9 million fulltext articles, only 2.7 million are part of the "[Open Access Subset](https://www.ncbi.nlm.nih.gov/pmc/tools/openftlist/)" which allows for downloading the structured fulltext as opposed to just viewing the article online.
+Luckily however, since we only cared about authorship information, we were able to download structured frontmatter (rather than fulltext) records from PMC's [OAI-PMH service](https://www.ncbi.nlm.nih.gov/pmc/tools/oai/), such that we were not limited to just the OA Subset.
+For PubMed, we used the E-Utilities APIs.
+For PubMed records, we were able to extract author names (first and last) and order.
+For PMC, we were able to extract these fields as well as whether each author was a corresponding author.
+To automate and generalize these tasks, we created the [pubmedpy](https://github.com/dhimmel/pubmedpy) Python package.
+
+We selected three journals to represent the field of bioinformatics and computational biology, including two ISCB Partner Journals (_PLOS Computational Biology_ and _Bioinformatics_) and one field-specific journal that is not a partner (_BMC Bioinformatics_).
+From PubMed, we compiled a catalog of 29,755 journal articles published from when each journal was established through 2019.
+We were able to retrieve authorship information for all but 6 of these articles using PubMed or PubMed Central.
+
+To determine corresponding authors for an article, we relied on PMC data if available (20,696 articles) and otherwise fellback to PubMed data (9,053 articles).
+Almost all articles without PMC data were from _Bioinformatics_, since its a "selective deposit" rather than "full paricipation" journal in PMC.
+If coded corresponding author status was not available (120 articles from PMC and all articles from PubMed), we assumed the last author was corresponding.
+
+We performed further analysis on PMC authors to learn more about corresponding author practices.
```

Comment:
The first part of this paragraph is a bit resultsy but it's more of a justification for the latter part of the paragraph, so I think it's fine. Someone may want us to change where it is but we can cross that bridge if we get there.